mpiopenacc/readme.cce

```diff
@@ -33,10 +39,14 @@ ldd mpiacc_f.exe | grep "not"
 
 ### C
 cc -hacc ../src/mpiacc_c.c -o mpiacc_c.exe
+# CC-1254 crayc: WARNING 
+#  The environment variable "CPATH" is not supported.
```

Comment:
GCC/4.9.3-binutils-2.25 defines the CPATH variable and the Cray compiler apparently doesn't like it. 

Normally this warning should be harmless, if you get missing std c++ headers we might need to use the craypkgconfig tool to make sure that the Cray wrapper is using the correct include path when using this GCC module.

If needed, we might also consider removing this CPATH environment from the GCC module.